testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/theme/ExtendingThemeTest.java

```diff
@@ -66,4 +84,43 @@ public void environmentVariablesSubstitutionInThemeProperties() {
             }
         });
     }
+
+    // KEYCLOAK-11729
+    // If 'shouldBeInContainer' is true, then extending theme will be copied to JBoss-based container.
+    // Otherwise, the theme will be deleted from JBoss-based container (clean-up).
+    private void themeToContainer(boolean shouldBeInContainer) {
```

Comment:
Is it rather possible to copy the theme at the time when auth-server-wildfly (or auth-server-eap) is prepared? Possibly here https://github.com/keycloak/keycloak/pull/6410 ? That could be compliant with existing tests, which use some similar functionality and require to have some files available on auth server side.

Reply:
Thanks for advice!